vendor/src/github.com/docker/engine-api/types/registry/registry.go

```diff
@@ -82,7 +82,7 @@ type SearchResult struct {
 	IsOfficial bool `json:"is_official"`
 	// Name is the name of the repository
 	Name string `json:"name"`
-	// IsOfficial indicates whether the result is trusted
+	// IsTrusted indicates whether the result is trusted
```

Comment:
This should not be called `IsTrusted`. This field has no bearing on the trust state of an image. Not sure if we need a PR in engine-api to handle this.